MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+recursive-include openfisca_web_api/i18n *.mo
+include *.md
+include LICENSE
+include *.ini
```

Comment:
Thanks for the source link!

I need to dive into the question before merging the pull request. Thanks again for it (for the 4 actually).